src/lib.rs

```diff
@@ -31,7 +74,7 @@ use proc_macro::TokenStream;
 ///
 /// Output AST:
 ///
-/// ```no_test
+/// ```ignore
```

Comment:
```suggestion
/// ```no_run
/// # #[cfg(doc)]
```